docs/build/build-hrmp-channels.md

```diff
@@ -59,40 +59,20 @@ of the `XcmExecutor`, which handles notifications of channel requests and accept
 
 ## Opening HRMP Channels with System Parachains
 
-HRMP channel management involving system parachains takes place entirely on the Relay Chain. No
-action is required from the parachain origin.
+The [`establish_channel_with_system`](https://paritytech.github.io/polkadot-sdk/master/polkadot_runtime_parachains/hrmp/pallet/dispatchables/fn.establish_channel_with_system.html)
+call can be used to establish a bi-directional channel between any parachain and a system chain.
+This call needs to be dispatched from the parachain via an XCM program to execute on the Relay Chain.
 
-Opening an HRMP channel with a system parachain requires an
-[OpenGov referendum](../learn/learn-guides-polkadot-opengov.md) using the
-[General Admin Track](../learn/learn-polkadot-opengov-origins.md#general-admin).
+For instance, the call to be executed on the relay chain to establish bi-directional channel between
+a parachain and the Asset Hub with `ParaID` 1000 is `0x3c0ae8030000`
 
-Proposals should generally be a `batchAll` call containing two `forceOpenHrmpChannel` calls (one for
-each direction of the channel).
+![asset-hub-hrmp-channel](../assets/asset-hub/establish-channel-asset-hub.png)
 
-As an example, see [Referendum 280](https://polkadot.polkassembly.io/referenda/280), which opened
-two bi-directional channels with Asset Hub (one for Zeitgeist and one for Composable).
+Here is an example call with XCM message sent from the parachain to Rococo relay
+chain to establish channel a bi-directional channel with Rococo Asset Hub -
+`0x0f001f000301000314000400000000070010a5d4e81300000000070010a5d4e80006000300c16678419c183c0ae8030000140d01000001003145`
```

Comment:
Not sure when I try to decode in PJS, it throws an error: 

![Screenshot 2024-08-05 at 12 10 49 PM](https://github.com/user-attachments/assets/f5ba0d14-6cc2-4fac-9868-766295864377)

Reply:
![image](https://github.com/user-attachments/assets/c19c8c3c-567c-46b8-b27e-077675ebb518)

A d was flipped to 0. Not sure how.

Reply:
`0x0f001f000301000314000400000000070010a5d4e81300000000070010a5d4e80006000300c16678419c183c0ae8030000140d01000001003145`

This is the actual call and it should work. Please try with this and merge this PR is it works.